src/components/forms/Button/styles.js

```diff
@@ -41,5 +42,16 @@ export default theme => ({
     '&::-moz-focus-inner': {
       border: 'none'
     }
+  },
+  arrow: {
+    display: props => (props.showArrow ? 'block' : 'none'),
+    position: 'absolute',
+    right: -2,
+    top: -2,
+    height: 'calc(100% + 4px)',
```

Comment:
Use flexbox to set its width and height. The two buttons should have the same size.